evennia/commands/default/building.py

```diff
@@ -205,7 +206,8 @@ def func(self):
         caller = self.caller
         args = self.args
         if not args:
-            caller.msg("Usage: @copy <obj> [=new_name[;alias;alias..]][:new_location] [, new_name2...]")
+            caller.msg("Usage: @copy <obj> [=<new_name>[;alias;alias..]]["
+                       ":<new_location>] [, <new_name2>...]")
```

Comment:
See below; I believe the lone `[` belongs on the next line in the case of a user command string example.